legal-api/src/legal_api/services/filings/validations/continuation_in.py

```diff
@@ -165,3 +169,22 @@ def validate_continuation_in_court_order(filing: dict, filing_type) -> list:
         if err:
             return err
     return []
+
+
+def validate_business_in_colin(filing_json: dict, filing_type: str) -> list:
+    """Validate continuation EXPRO business by making a call to Colin API."""
+    msg = []
+    business_identifier_path = f'/filing/{filing_type}/business/identifier'
+
+    if filing_json['filing'][filing_type].get('business'):
+        identifier = filing_json['filing'][filing_type]['business']['identifier']
+        url = f'{current_app.config["COLIN_URL"]}/businesses/{identifier}/public'
+        headers = {
+            'Authorization': f'Bearer {jwt.get_token_auth_header()}',
+            'Content-Type': 'application/json'
+        }
+        response = requests.get(url, headers=headers)
```

Comment:
Do we need to check anything else other than Business was found?

Reply:
may be you can compare the legal name (https://github.com/bcgov/lear/blob/main/queue_services/entity-filer/src/entity_filer/filing_processors/continuation_in.py#L52)

Reply:
Can you create a new service `colin.py` and move this api call to services folder (similar to namex)